content/200-concepts/300-more/050-environment-variables/200-using-multiple-env-files.mdx

```diff
@@ -51,14 +88,6 @@ Any top level script that is running the tests and migrations needs the dotenv c
 
 </Admonition>
 
-### Migration script
-
-```json file=package.json
-  "scripts": {
-    "migrate:mysql": "dotenv -e .env.test -- npx prisma migrate dev --name mysql-init",
-  },
-```
```

Comment:
If you rewrite this, it might actually be a good example how you can make sure the Prisma CLI can get different environments (test vs. produciton) when running `prisma migrate dev` - that makes a _lot_ of sense.

In general you probably want to have 1 example for Prisma Client, and one for Prisma CLI/Migrate here to show the actual different use cases. The Jest example might indeed be the one for Prisma Client - but make it explicit.